ksql-engine/src/main/java/io/confluent/ksql/KsqlContext.java

```diff
@@ -17,8 +17,8 @@
 package io.confluent.ksql;
 
 import io.confluent.ksql.metastore.MetaStore;
-import io.confluent.ksql.util.KafkaTopicClientImpl;
-import io.confluent.ksql.util.KsqlConfig;
+import io.confluent.ksql.util.*;
```

Comment:
you probably need to change your import settings?

Reply:
err - yep - will do